packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -204,6 +205,9 @@ async function createProjectFiles(newAppDir, overwrite, yarn1) {
     path.join(newAppDir, '.gitignore')
   )
 
+  // Write the uid
+  fs.writeFileSync(path.join(newAppDir, '.redwood', 'telemetry.txt'), UID)
```

Comment:
You probably want ensureDirSync from `fs-extra` which I think is ok to bring in here

Reply:
I assumed because we'd just copied the template and it includes the folder I'd be able to get away with just assuming it existed?

Reply:
Ah nevermind I see the broken CI. Wilco but might just do a manual check for the sake of not bringing in the package.